pkg/restapi/operation/didexchange/models/invitation.go

```diff
@@ -24,6 +24,31 @@ type GenericError struct {
 	} `json:"body"`
 }
 
+// CreateInvitationRequest model
+//
+// This is used for operation to create invitation
+//
+// swagger:parameters createInvitation
+type CreateInvitationRequest struct {
+	// Params for creating invitation
+	//
+	// in: path
+	*CreateInvitationParams
+}
+
+// CreateInvitationParams model
+//
+// This is used for creating invitation
+//
+type CreateInvitationParams struct {
+
+	// The Alias to be used in invitation to be created
+	Alias string `json:"alias"`
```

Comment:
Is Alias the same as Label?

Reply:
I'm confused by this code: https://github.com/hyperledger/aries-framework-go/pull/614/files#diff-5213b774cf71e1cef26c7bb7c48ee94eR110

Reply:
Removed that code